lib/analytics_events_documenter.rb

```diff
@@ -19,6 +19,7 @@ class AnalyticsEventsDocumenter
     pending_profile_idv_level
     proofing_components
     profile_history
+    recaptcha_annotation
```

Comment:
Is this to avoid errors on missing documentation for the analytics method call? I'd expect we'd update the analytics method signatures to include this property if it's something we expect will be logged.

Reply:
IMO, the `recaptcha_annotation` is of lesser interest in the events being logged and I was leveraging the `**extra` argument in the signatures of all the affected events.

If we think it is important and want to add the documentation, I can do so.

Reply:
I think it's relevant to be documented.